### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jwulf**
Add programmatic use example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,37 @@ npm install --save ts-json-schema-generator
     Do not sort properties.
 ```
 
+## Programmatic Usage
+
+Here is an example of using it programmatically:
+
+```typescript
+import { SchemaGenerator } from "ts-json-schema-generator";
+import * as ts from "typescript";
```
Comment (domoritz) [inline README.md]: Does typescript have a default export?

File: README.md [hunk 2]
```diff
@@ -51,6 +51,37 @@ npm install --save ts-json-schema-generator
...
+import { createFormatter } from "ts-json-schema-generator/dist/factory/formatter";
+import { createParser } from "ts-json-schema-generator/dist/factory/parser";
+import { createProgram } from "ts-json-schema-generator/dist/factory/program";
+import { Config } from 	"ts-json-schema-generator/dist/src/Config";
```
Comment (domoritz) [inline README.md]: We should instead expose these from the index.js file so you don't have to import from specific files.

File: README.md [hunk 3]
```diff
@@ -51,6 +51,37 @@ npm install --save ts-json-schema-generator
...
+  };
+	 const program: ts.Program = createProgram(config);
+
+	 const generator: SchemaGenerator = new SchemaGenerator(
+		  program,
+		  createParser(program, config),
+		  createFormatter(config),
+	 );
+ 	const schema = generator.createSchema(type);
+	 return schema;
```
Comment (domoritz) [inline README.md]: Fix indentation

File: README.md (hunk 1 above)
Comment (jwulf) [reply README.md]: Must do. I got this code from the tests in the project, and it works for me. I'm using it in production.

File: README.md (hunk 2 above)
Comment (jwulf) [reply README.md]: That would be good. It does work like this now.

File: README.md (hunk 3 above)
Comment (jwulf) [reply README.md]: Updated. Hopefully fixed, now.

File: README.md (hunk 1 above)
Comment (domoritz) [reply README.md]: So `inport ts from 'typescript'` does not work.

File: README.md (hunk 2 above)
Comment (domoritz) [reply README.md]: Let's make it work so that the usage is clean.

File: README.md (hunk 1 above)
Comment (jwulf) [reply README.md]: Yeah, it does work. Intellisense suggested it to me as a refactor.

File: README.md (hunk 2 above)
Comment (domoritz) [reply README.md]: We are exposing all of this from `factory/index.ts`.

**Comment by domoritz** [comment]
Please open a new PR.

**Pull request closed without merging**